Source/CarthageKit/Xcode.swift

```diff
@@ -228,6 +228,12 @@ public func xcodebuildTask(task: String, _ buildArguments: BuildArguments) -> Ta
 	return Task("/usr/bin/xcrun", arguments: buildArguments.arguments + [ task ])
 }
 
+/// Creates a task description for executing `xcodebuild` with the given
+/// arguments.
+public func xcodebuildTask(task: String, _ task2: String, _ buildArguments: BuildArguments) -> Task {
+	return Task("/usr/bin/xcrun", arguments: buildArguments.arguments + [ task, task2 ])
```

Comment:
This implementation is repeated from the other `xcodebuildTask` overload. What do you think about making the canonical implementation take an array of `tasks` instead? Then we can leave the overloads for simplicity.